lib-gdal/src/main/java/org/esa/snap/dataio/gdal/GDALInstaller.java

```diff
@@ -250,16 +306,19 @@ private static void copyDistributionArchiveAndInstall(Path gdalDistributionRootF
             logger.log(Level.FINE, "The distribution root folder '" + gdalDistributionRootFolderPath + "' exists on the local disk.");
         }
         final Path zipFilePath = gdalVersion.getZipFilePath();
-        try {
-            if (logger.isLoggable(Level.FINE)) {
-                logger.log(Level.FINE, "Copy the zip archive to folder '" + zipFilePath + "'.");
-            }
-            final URL zipFileURLFromSources = gdalVersion.getZipFileURLFromSources();
-            if (zipFileURLFromSources == null) {
-                throw new ExceptionInInitializerError("No GDAL distribution drivers provided for this OS.");
-            }
-            FileHelper.copyFile(zipFileURLFromSources, zipFilePath);
+        if (logger.isLoggable(Level.FINE)) {
```

Comment:
If(isLoggable()) is not necessary.
This is the first call in Logger.log()

Reply:
Why is it necessary to compute the hash. Don't we know which GDAL version we have installed?

Reply:
> Why is it necessary to compute the hash. Don't we know which GDAL version we have installed? Or is it done to ensure that everything is copied correctly? But I wonder why this is necessary. Usually extracting auxdata or libs from jars simply works. Is this case somehow special?

Is it done to ensure that everything is copied correctly.

Argument:
In an unknown conditions, the internal GDAL distribution it is not installed correctly (some files are missing) which cause error at loading time or using time. We cannot reproduce these problems, but are trying this support solution. It verifies the installation integrity using the hash of the internal GDAL distribution directory after extraction from ZIP. If the integrity verification fails, a runtime exception is thrown which is logged in the log file. The user will know the reason  why the GDAL native libraries are not loaded, and he can restart the SNAP to retry the installation of internal GDAL distribution.

Reply:
Okay, thanks